docs/auth/oidc/provider.md

```diff
@@ -0,0 +1,116 @@
+
+It is assumed an IAP is already handling traffic for Backstage.
+
+## Configuration
+
+Start by adding the following `auth` configuration in your `app-config.yaml` or
+`app-config.production.yaml` or similar:
+
+```yaml
+auth:
+  providers:
+    oidc-proxy:
+      # issuer is required, must match the iss claim of the id token.
+      issuer: https://login.example.com
+      # audience is required, must match one of the aud values listed in the id token.
+      audience: 'https://backstage.example.com'
+      # oidcIdTokenHeader is optional and may be set to any value except `authorization`.
+      # The value of the header is the id token without any prefix or suffix.
+      oidcIdTokenHeader: 'x-oidc-id-token'
+      signIn:
+        resolvers:
+          # emailMatchingUserEntityProfileEmail resolves the authenticated id
+          # token to a catalog entity by matching the email token claim to the
+          # User spec.profile.email field of the catalog entity.
+          - resolver: emailMatchingUserEntityProfileEmail
+          # signInWithoutCatalogUser allows Sign-In without a pre-existing User
+          # entity in the catalog.
+          - resolver: signInWithoutCatalogUser
+```
+
+Consult your identity provider to obtain the value of the `issuer` and `audience` claims. If you have a token you can extract these values using [jwt.io](https://jwt.io).
+
+This config section must be in place for the provider to load at all. Now let's add the provider itself.
+
+### Resolvers
+
+The `oidcProxy` provider supports two resolvers.
+
+- `emailMatchingUserEntityProfileEmail`: Matches the `email` id token claim with the User entity that has a matching `spec.profile.email`.
+- `signInWithoutCatalogUser`: allows Sign-In without a pre-existing User entity in the catalog.
+
+:::note Note
+
+The resolvers will be tried in order.
+
+:::
+
+If these resolvers do not fit your needs you can build a custom resolver, this is covered in the [Building Custom Resolvers](../identity-resolver.md#building-custom-resolvers) section of the Sign-in Identities and Resolvers documentation.
+
+## Backend Changes
+
+There is a module for this provider that you will need to add to your backend.
+
+First you'll want to run this command to add the module:
+
+```sh
+yarn --cwd packages/backend add @backstage/plugin-auth-backend-module-oidc-proxy-provider
+```
+
+Then you will need to add this to your backend:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+backend.add(import('@backstage/plugin-auth-backend'));
+/* highlight-add-start */
+backend.add(
+  import('@backstage/plugin-auth-backend-module-oidc-proxy-provider'),
+);
+/* highlight-add-end */
+```
+
+## Frontend Changes
+
+This provider is intended for use with the `ProxiedSignInPage`, which is installed in `packages/app/src/App.tsx` like this:
```

Comment:
could also link to https://backstage.io/docs/auth/#sign-in-with-proxy-providers instead, which will be updated once the new frontend system is in place and the method of installing this sign in page changes.

Reply:
Fixed in 5ee2b1ea6eee `oidc-proxy-provider: link to sign-in with proxy providers`